### PR TITLE
Adds MongoDB Connection

### LINKED ISSUE #96: [2pts] App Backend: Connect MongoDB to Server
**Fill in the blanks**:
As a developer, I would like to store data on the DB, so that data is stored.

**Describe the solution you'd like:**
- [x] create mongoDB object that is interactable

**Definition of Done - Check as you go!**
- [x] Tasks have been implemented
- [x] Documented
- [x] Passing Tests for each task
- [x] Pull Request made
- [ ] Code Reviewed

### PR DESCRIPTION
**Pull request by RafaelPiloto10**

### CODE REVIEW

**Review by brendacano — COMMENTED** [review]
How do you know how to do this? Like, what did you look up or use?

**Comment by RafaelPiloto10** [comment]
> How do you know how to do this? Like, what did you look up or use?

I just googled "python mongodb connect" and used this site: https://www.mongodb.com/languages/python

File: app_backend/database.py
Comment (MimiOlayeye) [inline app_backend/database.py:1]: Maybe you could add some comments if you want (most of the variables are self-explanatory though), but otherwise, it looks good.

File: app_backend/server_test.py
Comment (MimiOlayeye) [inline app_backend/server_test.py:12]: The friends feature is cool! How did u test this?

**Review by MimiOlayeye — APPROVED** [review]

File: app_backend/server_test.py
Comment (RafaelPiloto10) [reply app_backend/server_test.py:12]: Refer to #83

**Review by oreorz — COMMENTED** [review]
We may want to wait until David's backend restructure is merged first? Or no?

**Comment by DDVD233** [comment]
> We may want to wait until David's backend restructure is merged first? Or no?

I think it'd be better to merge this one first. I will handle the conflict

File: app_backend/database.py
Comment (DDVD233) [inline app_backend/database.py]: Is return needed here?

**Review by DDVD233 — APPROVED** [review]
Looks good to me. 

One package you might want to consider using is `motor`. It is a fast, non-blocking MongoDB client. It is recommended by [the official doc](https://www.mongodb.com/developer/quickstart/python-quickstart-fastapi/). 

For our project, I think both approaches work fine. Concurrency won't be a problem.

**Review by brendacano — APPROVED** [review]
looks good

**Pull request merged**
